Source/CarthageKit/VersionFile.swift

```diff
@@ -142,15 +146,16 @@ struct VersionFile {
 	}
 
 	func satisfies(platform: Platform, commitish: String, binariesDirectoryURL: URL, localSwiftVersion: String) -> SignalProducer<Bool, CarthageError> {
-		guard let cachedFrameworks = self[platform] else {
-			return SignalProducer(value: false)
-		}
+		guard let cachedFrameworks = self[platform] else { return SignalProducer(value: false) }
 
 		let hashes = self.hashes(for: cachedFrameworks, platform: platform, binariesDirectoryURL: binariesDirectoryURL)
 			.collect()
 
-		let swiftVersionMatches = self.swiftVersionMatches(for: cachedFrameworks, platform: platform, binariesDirectoryURL: binariesDirectoryURL, localSwiftVersion: localSwiftVersion)
-			.collect()
+		let swiftVersionMatches = self.swiftVersionMatches(
+			for: cachedFrameworks, platform: platform,
+			binariesDirectoryURL: binariesDirectoryURL, localSwiftVersion: localSwiftVersion
+		)
+		.collect()
```

Comment:
Again, same problem as above, how can this be indented if the line before is a `)`.